charts/trusted-artifact-signer/templates/tests/test-sign-verify.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: trusted-artifact-signer-test-sign-verify
+  labels:
+    app.kubernetes.io/component: trusted-artifact-signer
+  annotations:
+      helm.sh/hook: test
+spec:
+    initContainers:
+      - name: buildah
+        image: quay.io/buildah/stable
+        command: ["/bin/sh", "-c"]
+        args:
+        - |
+            buildah pull alpine:latest
+            buildah tag alpine:latest ttl.sh/sigstore-test:5m
+            buildah push ttl.sh/sigstore-test:5m
+        securityContext:
+            privileged: true
```

Comment:
Does this init container need to run as `privileged` just to pull, tag and push an image?

Reply:
I am afraid that the container is not able to pull image without privileged context
```
sh-5.2# buildah pull alpine:latest
Error: error writing "0 0 4294967295\n" to /proc/17/uid_map: write /proc/17/uid_map: operation not permitted
ERRO[0000] writing "0 0 4294967295\n" to /proc/17/uid_map: write /proc/17/uid_map: operation not permitted 
ERRO[0000] (Unable to determine exit status)   
```